srcpkgs/bc-gh/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'bc-gh'
 pkgname=bc-gh
-version=3.1.5
+version=3.3.3
 revision=1
 wrksrc="bc-${version}"
 short_desc="Implementation of POSIX bc with GNU extensions"
 maintainer="Gavin D. Howard <yzena.tech@gmail.com>"
 license="BSD-2-Clause"
 homepage="https://github.com/gavinhoward/bc"
+changelog="https://raw.githubusercontent.com/gavinhoward/bc/${version}/NEWS.md"
```

Comment:
Please use master instead of `${version}`, using a specific branch is necessary only if they use release branches.

Reply:
Done, thanks :)